config/manifests/bases/oran-o2ims.clusterserviceversion.yaml

```diff
@@ -257,6 +212,52 @@ spec:
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:text
       version: v1alpha1
+    - description: ProvisioningRequest is the Schema for the provisioningrequests
+        API
+      displayName: ORAN O2IMS Cluster Request
```

Comment:
Cluster -> Provisioning

Reply:
Right, there are a bunch of occurrences with the space included. I'll fix them asap. Thanks!

Reply:
Ok, done.